### DISCUSSION TITLE
Argument matching

### DISCUSSION THREAD

**Discussion by ojal** [seroanalytics/serosolver#57] [General]
Hi James and team,
I have seen that serosolver back on active development. I am currently using it for an analysis, many thanks for providing the package! it save a lot of coding from my side!. I have noted the argument matching is not working properly which results in the serosolver() function throwing errors. It might help meanwhile to put a note to user to ensure they name their arguments when parsing them to the function, rather than depend on the old sequence of arguments. 

Regards,
John.

**Comment by QiangWanglxb** [comment]
Yes. I think the same question happens to me either. Here are the errors reported in R:

Error in data.frame(x_coord = 1, y_coord = 1, inf_times = possible_exposure_times) : 
  arguments imply differing number of rows: 1, 0

**Reply by jameshay218** [reply]
Hi both,

So sorry for the very very slow reply. I didn't get a notification on this.

Are you still trying to use serosolver? I realise this is perhaps too late, but let me know if you are still interested in this and I can help resolve these issues.

Best,
James
